custom/benin/management/commands/migrate_users_and_their_cases_to_new_rc_level.py

```diff
@@ -192,10 +193,11 @@ def _find_case_ids(case_type, owner_id, opened_by_user_id):
     )
 
 
-def _update_users_location(user, location, dry_run):
+def _update_users_location(user, existing_location, new_location, dry_run):
     if not dry_run:
-        user.set_location(location)
-    log(f"User {user.username}:{user.user_id} location updated to {location.location_id}")
+        user.set_location(new_location)
+        user.unset_location_by_id(existing_location.location_id)
```

Comment:
oh this was needed?

Reply:
Yes, based on the discussion with Adam, it was decided to have only the `rc` location for the mobile worker.
(Side note: It also helps issues (location ambiguity) with Case Sharing to have only 1 location that own cases)

Reply:
okay, good to not do this if `existing_location` is same as `new_location`, Not sure if that is possible or not.

Reply:
`new_location` will always be different from `existing_location` for this script.